lib/rules/time-no-imperceptible/README.md

```diff
@@ -1,5 +1,7 @@
 # time-no-imperceptible
 
+***Deprecated: this rule is outside the scope of stylelint's functionality. See [the release planning docs](https://stylelint.io/user-guide/release-planning/) for details.***
```

Comment:
Awesome. Let's update this message to:

***Deprecated: use the `time-min-milliseconds` rule with `100` as its primary option. ***